docs/Architecture.md

```diff
@@ -284,7 +284,19 @@ To support development and test of the Strimzi UI, a number of tools and utiliti
 
 #### Storybook
 
-This section to follow in a future PR/as a part of https://github.com/strimzi/strimzi-ui/issues/14 .
+[Storybook](https://storybook.js.org/) is a UI development tool that helps to build UI components in isolation and record their states as stories. This allows separation of concerns and ensures components are developed in a composable manner. Additionally, Storybook helps document components for reuse, can be used for visual testing of components and can also be extended with an [ecosystem of addons](https://storybook.js.org/addons).
+
+When developing [Element](../client/Elements), [Group](../client/Groups) or [Panel](../client/Panels) components for the Strimzi UI, Storybook stories are implemented alongside the component code, allowing reviewers to easily run the component, see the visual rendering of the component and manually test the functionality. As referenced in the [file name conventions](#file-name-conventions) section, Storybook stories are defined in separate files in the same directory as the component implementation files, and should be suffixed with `.stories.js`.
+
+By building up a library of Storybook stories, all the UI components in the Strimzi UI can be viewed in one place with minimal effort, allowing easy composition and creation of higher-level components. If possible, both view layers (`CARBON` and `PATTERNFLY`) will be viewable when running Storybook, allowing developers to easily compare, contrast and ensure equality of functionality when developing for either view layer. If it is not possible to run a Storybook with both view layers, a separate Storybook instance can be run for each view layer.
```

Comment:
I think we could support both in one storybook by, rather than using an alias, reference the individual view layer implementations/importing them. I would hope we could have a common helper/knob for this, which is available across all stories. Out of interest, have you tried (even in a mock sense) anything like this?

Reply:
Updated this line. Haven't tried anything like this yet.